rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -284,13 +282,11 @@ def run(self):
         A list of `rocalTensorList` objects for respective pipeline outputs.
         """
         try:
-            print("getRemainingImages :", self.getRemainingImages())
-            if self.getRemainingImages() > 0:
-                self.rocalRun()
+            if self.get_remaining_images() > 0:
+                self.rocal_run()
                 return b.getOutputTensors(self._handle)
         except:
-                print("Raise stop iter")
-                raise StopIteration
+            raise StopIteration
 
 
 def _discriminate_args(func, **func_kwargs):
```

Comment:
@swetha097 From where we took reference for this function?

Reply:
We have taken reference for this from DALI. Rajy has added this for pipeline decorators

Reply:
@SundarRajan28  @swetha097 We are not supposed to use directly from DALI. Please modify the function names , comments and other variables.